aiida/orm/groups.py

```diff
@@ -340,21 +340,12 @@ def get(cls, **kwargs):
         """
         from aiida.orm import QueryBuilder
 
-        filters = {}
         if 'type_string' in kwargs:
+            message = '`type_string` is deprecated because it is determined automatically'
+            warnings.warn(message)  # pylint: disable=no-member
             type_check(kwargs['type_string'], str)
 
-        query = QueryBuilder()
-        for key, val in kwargs.items():
-            filters[key] = val
-
-        query.append(cls, filters=filters)
-        results = query.all()
-        if len(results) > 1:
-            raise exceptions.MultipleObjectsError("Found {} groups matching criteria '{}'".format(len(results), kwargs))
-        if not results:
-            raise exceptions.NotExistent("No group found matching criteria '{}'".format(kwargs))
-        return results[0][0]
```

Comment:
Are these modifications related to the topic? Is it even doing the same thing? At least it would seem that some exceptions would be missed, no?

Reply:
It would be, except that if you look at the behavior of `QueryBuilder.one()` already has exactly the same behavior, so this code that I removed was just redundant. The behavior should remain exactly the same. If you think it doesn't, let me know, because then I messed up ;)

Reply:
Ah, I see, yes, thats true. The only difference is in the second check `if not results` vs `elif len(res) == 0`, but I can't think of a case where this would cause a divergent behaviour, let alone one which would actually be problematic. I mean, if res/result was a float let's say, it would have problems when evaluating `elif len(res) == 0` but would pass `if not results`, although in that case it would already had problems with the first `if len(results) > 1`. So no, I guess no complaints here.